### PR TITLE
Feat/beamer injected

### PR DESCRIPTION
**Pull request by maurodibert**
# Flutter PRs

| :point_up: don't even dare to be here if you didn't check [before_pr.md](./docs/before_pr.md) file |
| --- |

Thank you for adding your efforts to this awesome project and Flutter Team! 🤘
Please, keep your PRs short to make everybody happy
_made with love by [Mau](https://github.com/maurodibert) and [Martin](https://github.com/mal2tin)  for the_ **Monkslab Team** ❤️

## Keep it clean

These are our selected best practices that we agree to follow for any PR to be approved. We think it's super handy for including everything without having to think about it. We hope they will make your life easier! 🙏

## Type

- [x] New feature
- [ ] Testing
- [ ] Refactor
- [ ] Bug-fix

## Versions

- [x] upgrade with [Semantic Versioning](https://semver.org/) in pubspec.yaml: **0.0.0+4**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +70,7 @@ main
 
 ## :point_up: Decisions and Considerations
 
+- **follow [our patterns and conventions](./docs/conventions.md)**
```
Comment (EugenioTesio) [inline README.md:73]: Great!!

File: docs/1_after_pr.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
-# After PR
+# 🧙 After PR
 
-_made with love by [Mau](https://github.com/maurodibert) and [Martin](https://github.com/mal2tin)  for the_ **Monkslab Team** ❤️
+_Made with love by the_ **Monkslab** ❤️
```
Comment (EugenioTesio) [inline docs/1_after_pr.md:4]: Go MonksLab go..

**Review by EugenioTesio — COMMENTED** [review]
Check my comment throughout the code.

File: README.md (hunk 1 above)
Comment (maurodibert) [reply README.md:73]: Yeah!

File: docs/1_after_pr.md (hunk 2 above)
Comment (maurodibert) [reply docs/1_after_pr.md:4]: Yeah again!

File: README.md (hunk 1 above)
Comment (EugenioTesio) [reply README.md:73]: Ok, well done!!

**Review by EugenioTesio — APPROVED** [review]

**Pull request merged**